libraries/ui-shared/Checkbox/index.jsx

```diff
@@ -13,13 +14,24 @@ import styles from './style';
  * @param {Object} props The component properties.
  * @returns {JSX}
  */
-const Checkbox = ({ checkedClassName, unCheckedClassName, ...props }) => (
-  <BaseCheckbox
-    {...props}
-    checkedIcon={<CheckedIcon className={classNames(styles.icon, checkedClassName)} />}
-    uncheckedIcon={<UncheckedIcon className={classNames(styles.icon, unCheckedClassName)} />}
-  />
-);
+const Checkbox = ({ checkedClassName, unCheckedClassName, ...props }) => {
+  const { checkbox: { icon: { checked = {}, unchecked = {} } = {} } = {} }
```

Comment:
Those context values are static. Why do we need to contextualize them ?

Reply:
removed context and added simple class names to native dom elements